static/js/hotkey.js

```diff
@@ -256,6 +260,23 @@ function process_hotkey(e) {
         }
     }
 
+    if (event_name === 'left_arrow') {
+        var msg = current_msg_list.get_last_own_editable_message();
+        if (msg) {
+            var msg_row = current_msg_list.get_row(msg.id);
+            current_msg_list.select_id(rows.id(msg_row), {then_scroll: true, from_scroll: true});
+            message_edit.start(msg_row);
+            var editability_type = message_edit.get_editability(msg, 5);
+            setTimeout(function () {
+                if (editability_type === message_edit.editability_types.TOPIC_ONLY) {
+                    ui.focus_on('message_edit_topic');
+                } else {
+                    ui.focus_on('message_edit_content');
+                }
+            }, 300);
```

Comment:
What's the story with the setTimeout here?  Is it to wait for the message edit window to open?  We can't hardcode a wait time, since whether that works will depend on the RTT.  Instead, we should do this correctly: pass an option to `message_edit.start`, and that function's success handler should call `ui.focus_on` appropriately.